### PR TITLE
Stacked bar chart interaction changes and css modifications

### PR DESCRIPTION
**Pull request by nulikartik**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ npm run change`

#### Description of changes

Changing interaction behavior between legends and multi-stacked and single stacked bar chart. Changing styling and opacity for interactions.

ISOLATION state:
image

#### Focus areas to test

(optional)

### CODE REVIEW

File: packages/charting/src/components/StackedBarChart/MultiStackedBarChart.base.tsx [hunk 1]
```diff
@@ -107,12 +112,14 @@ export class MultiStackedBarChartBase extends React.Component<IMultiStackedBarCh
       }
       value = (pointData / total) * 100;
       startingPoint.push(prevPosition);
-      const isSelected = this.state.legendSelected === point.legend!;
       const styles = this.props.styles;
+      let shouldHighlight = true;
+      if (this.state.isLegendHovered || this.state.isLegendSelected) {
+        shouldHighlight = this.state.selectedLegendTitle === point.legend ? true : false;
```
Comment (kkjeer) [inline packages/charting/src/components/StackedBarChart/MultiStackedBarChart.base.tsx]: Nit: unnecessary ternary operator, can just be `this.state.selectedLegendTitle === point.legend`

File: packages/charting/src/components/StackedBarChart/StackedBarChart.base.tsx [hunk 2]
```diff
@@ -148,12 +153,14 @@ export class StackedBarChartBase extends React.Component<IStackedBarChartProps,
       }
       value = (pointData / total) * 100;
       startingPoint.push(prevPosition);
-      const isSelected = this.state.legendSelected === point.legend!;
       const styles = this.props.styles;
+      let shouldHighlight = true;
+      if (this.state.isLegendHovered || this.state.isLegendSelected) {
+        shouldHighlight = this.state.selectedLegendTitle === point.legend ? true : false;
```
Comment (kkjeer) [inline packages/charting/src/components/StackedBarChart/StackedBarChart.base.tsx]: Nit: unnecessary ternary operator (see above)

Comment (nulikartik) [reply packages/charting/src/components/StackedBarChart/StackedBarChart.base.tsx]: Removed it.

File: packages/charting/src/components/StackedBarChart/MultiStackedBarChart.base.tsx (hunk 1 above)
Comment (nulikartik) [reply packages/charting/src/components/StackedBarChart/MultiStackedBarChart.base.tsx]: removed the ternary operator.

**Pull request merged**